build.override.targets

```diff
@@ -3,37 +3,49 @@
 
   <PropertyGroup>
     <TraversalBuildDependsOn>
+      WcfCopyProjectJson;
       WcfSetup;
       $(TraversalBuildDependsOn);
       WcfCleanup;
     </TraversalBuildDependsOn>
   </PropertyGroup>
-  
+
   <PropertyGroup>
     <TraversalCleanDependsOn>
       WcfClean;
       $(TraversalCleanDependsOn);
     </TraversalCleanDependsOn>
   </PropertyGroup>
-  
+
   <PropertyGroup>
     <WcfSelfHostSetupLog>$(MSBuildThisFileDirectory)\SelfHostedWcfServiceSetup.log</WcfSelfHostSetupLog>
     <WcfSelfHostCleanupLog>$(MSBuildThisFileDirectory)\SelfHostedWcfServiceCleanup.log</WcfSelfHostCleanupLog>
   </PropertyGroup>
-  
+
+  <PropertyGroup>
+    <UseLatestDependencies Condition="'$(UseLatestDependencies)'==''">false</UseLatestDependencies>
+  </PropertyGroup>
```

Comment:
@iamjasonp raises a good point.  The normal MsBuild convention for boolean properties is that blank means false and that you should only really every check "== 'true'".  I think there is value having the entry here showing this property exists and its default value is false.  But we should not rely on matching the string 'false' anywhere.

---
In reply to: [72494232](https://github.com/dotnet/wcf/pull/1399#discussion_r72494232) [](ancestors = 72494232)